packages/atst/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@eth-optimism/atst",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "src/index.ts",
```

Comment:
Should all the files go under /dist? asking in relation to how we ship this, we'll we have `files` specified in the package.json or we'll be shipping both src and dist?

Reply:
shipping both src and dist.   Src is the source of truth for types.   Doing it this way is better devx because `go-to-definition` will take the user to the actual source code instead of a .d.ts file